include/CIndexStoreDB/CIndexStoreDB.h

```diff
@@ -215,6 +217,27 @@ indexstoredb_delegate_event_get_kind(_Nonnull indexstoredb_delegate_event_t);
 INDEXSTOREDB_PUBLIC
 uint64_t indexstoredb_delegate_event_get_count(_Nonnull indexstoredb_delegate_event_t);
 
+/// Valid only if the event kind is \p INDEXSTOREDB_EVENT_UNIT_OUT_OF_DATE, otherwise returns null.
+INDEXSTOREDB_PUBLIC _Nullable indexstoredb_unit_info_t
+indexstoredb_delegate_event_get_outofdate_unit_info(_Nonnull indexstoredb_delegate_event_t);
```

Comment:
Doc comment should mention the ownership of the unit info -- e.g.
```swift
/// The indexstoredb_unit_info_t pointer has the same lifetime as the \c indexstoredb_delegate_event_t
```

Reply:
Done.